.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I don't think we need this configuration. The VS Code CMake extension provides an easy way to launch debugging. Am I missing something here?

Reply:
Do we need any configuration file to setup the debugger for the CMake extension or does it work out of the box?

Reply:
No, it should work out of the box.

Reply:
Maybe tomorrow you can show me how this works in the extension because I do not see anything in VSCode nor did I find anything only about setting up code debugger with the CMake extension.